src/rovr_common/launch/sensors/aruco_localization.launch

```diff
@@ -1,27 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <launch>
-  <!-- args to be passed in -->
   <arg name="device" default="/dev/video0" />
   <arg name="show" default="true" />
-
-  <node pkg="aruco_localization" type="aruco_localization" name="aruco" output="screen">
+<arg name="show" default = "true"/>
+<node pkg="tf" type="static_transform_publisher" name="realsense_to_aruco" args="0 0 0 0 0 0 1 realsense_d415_link aruco 100" />
```

Comment:
What's the point of this static transform? If I understand correctly, this is a transform between a static object (`aruco`) and something that's on the robot (the realsense). Either way, it shouldn't be in this launch file, as all necessary links are managed elsewhere.